base/w3ctime.hpp

```diff
@@ -0,0 +1,44 @@
+#pragma once
+
+#include "std/array.hpp"
+#include "std/ctime.hpp"
+#include "std/string.hpp"
+
+// It would be better to use c++ standard means of
+// time parsing and unparsing like
+// time_get/time_put (<locale>) or get_time/put_time (<iomanip>).
+// But at the moment of writing they was not widely supporded
+// and/or worked inkorekd.
+//
+// All covertions are made in UTC
+
+namespace w3ctime
+{
+char constexpr kW3CTimeFormat[] = "%Y-%m-%dT%H:%MZ";
```

Comment:
This string will appear in each translation unit. Please, separate definition from declaration, move declaration to a source file.